tc/sgx/trusted_worker_manager/enclave/kme/CMakeLists.txt

```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+CMAKE_MINIMUM_REQUIRED(VERSION 3.2 FATAL_ERROR)
+PROJECT(libavalon-kme-enclave  CXX)
+
+INCLUDE(../CMakeVariables.txt)
+
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+# ADDITIONAL LIBRARIES
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+SET(TCF_TOP_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../../../")
+SET(TCF_CORE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../")
+#SET(WPE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/wpe)
+
+# XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+FILE(GLOB PROJECT_HEADERS *.h ../*.h)
+FILE(GLOB PROJECT_SOURCES *.cpp *.c ../*.cpp)
+FILE(GLOB PROJECT_EDL kme_enclave.edl)
+FILE(GLOB PROJECT_CONFIG *.xml)
+SOURCE_GROUP("Source" FILES ${PROJECT_HEADERS} ${PROJECT_SOURCES} ${PROJECT_EDL})
+
+SET(EDL_PATH ${CMAKE_CURRENT_SOURCE_DIR})
+
+INCLUDE_DIRECTORIES(${SGX_INCLUDE})
+INCLUDE_DIRECTORIES(/opt/intel/sgxsdk/include /opt/intel/sgxssl/include)
+INCLUDE_DIRECTORIES(${TCF_TOP_DIR}/common/cpp)
+INCLUDE_DIRECTORIES(${TCF_CORE_DIR}/trusted_worker_manager/common)
+INCLUDE_DIRECTORIES(${TCF_TOP_DIR}/common/cpp/crypto)
+INCLUDE_DIRECTORIES(${TCF_TOP_DIR}/common/cpp/verify_ias_report)
+INCLUDE_DIRECTORIES(${TCF_TOP_DIR}/common/cpp/packages/base64)
+INCLUDE_DIRECTORIES(${TCF_TOP_DIR}/common/cpp/packages/parson)
+INCLUDE_DIRECTORIES(${TCF_TOP_DIR}/common/sgx_workload/workload)
+INCLUDE_DIRECTORIES(${TCF_CORE_DIR}/trusted_worker_manager/enclave)
```

Comment:
Above directories are common for all KME, WPE and Singleton enclaves right ?

Reply:
Moved these statements to CMakeVariables.txt since they are common to all enclaves.